transmit-lib/src/cmr/transmit/metadata_db.clj

```diff
@@ -162,17 +162,25 @@
 
 (defn find-latest-concept
   "Searches metadata db for the latest concept matching the given parameters. Do not throw serivce
-  excpetion, returns the status and error message in a map in case of error."
+  exception, returns the status and error message in a map in case of error."
+  ([context params concept-type]
+   (find-latest-concept context params concept-type first))
+  ;"Allow for custom filtration of the results, instead of the default (first)"
+  ([context params concept-type filter-fn]
```

Comment:
What is the purpose of adding a filter to `find-latest-concept` rather than using `find-concepts`?

Reply:
By default, this function only returns the `(first...)` concept it finds, but for this purpose, we needed it to return all concepts it finds. So, the custom filter provided by the new `find-latest-concepts` is really just "no filter," although the filter could be more specific if ever needed for another purpose.

Reply:
Update on this comment and the one below: I have swapped to using `find-concepts`, so `find-latest-concept` will be reverted, and the new function will be removed.

Reply:
I have switched over to find-concepts - this is a much simpler solution, no arguments there.